doc/ems-application-guide/src/ems-actuators/thermal-envelope.tex

```diff
@@ -2,7 +2,7 @@ \section{Thermal Envelope}\label{thermal-envelope}
 
 \subsection{Window Shading Control}\label{window-shading-control}
 
-This actuator is available in models that also have the WindowProperty:ShadingControl input objects. The user-defined name for the WindowProperty:ShadingControl is not used to identify unique window shading controls; rather, the window name is used to identify the actuator. This is because there could be multiple windows, all with shades, each of which is governed by a single WindowProperty:ShadingControl input object. The EMS actuator can override the control actions for each window separately.
+Actuators called �Window Shading Control� are available with actuated component control type called �Control Status� in [ShadeStatus] and "Slat Angle" in [degree]. This actuator is available in models that also have the WindowProperty:ShadingControl input objects. The user-defined name for the WindowProperty:ShadingControl is not used to identify unique window shading controls; rather, the window name is used to identify the actuator. This is because there could be multiple windows, all with shades, each of which is governed by a single WindowProperty:ShadingControl input object. The EMS actuator can override the control actions for each window separately.
```

Comment:
The quotation marks around Window Shading Control aren't showing up during builds because they are invalid characters.  If this is the only problem I find along the way, I'll make the change.  If I find other changes, please make this along with them.